backend/infrahub/core/query/delete.py

```diff
@@ -0,0 +1,58 @@
+from typing import Any
```

Comment:
If this query is used only for testing purposes, I think it should be located under `tests`

Reply:
it's tricky because the goal is to expose it to other systems during integration tests so from Infrahub point of view it's not just for testing